README.md

```diff
@@ -1,8 +1,164 @@
+
+Sindri provides automated ZK proving infrastructure, empowering hundreds of teams—including leading Layer 2s and rollups—to launch in minutes instead of months.
+Through the Sindri API, developers can seamlessly integrate verifiable computation, reducing time to market, cutting costs, and scaling faster.
+Sindri makes zero-knowledge infrastructure simple and accessible, facilitating automation within the most hardware-intensive layer of the ZK app deployment stack.
+
+This repository hosts a proving application image and Helm chart which will route all zkEVM proof generation to the Sindri platform for a Scroll SDK chain.
+The following diagram depicts how the Scroll SDK components work together when you use Sindri as a prover.
+```mermaid
+---
+title: Scroll SDK with Sindri Provers
+---
+flowchart LR
+    subgraph settle[Settlement Layer]
+        direction LR
+    end
+    subgraph sequence[Sequencing Layer]
+        direction LR
+    end
+    subgraph prove[Proving Layer]
+        direction LR
+        C["**Coordinator**"] -.->|Tasks| P@{ shape: processes, label: "**Provers**\nThese components outsource proof generation to Sindri's GPU workers" }
+        P -.->|Proofs| C
+    end
+    subgraph sindri["Scroll Circuits on Sindri"]
+        direction TB
+        Chunk["**Chunk**
+        Verified execution of contiguous blocks (i.e. a zkEVM proof)"]
+        Batch["**Batch**
+        Intermediate aggregation of multiple chunks"]
+        Bundle["**Bundle**
+        Final aggregate proof sent to the settlement layer"]
+    end
+    settle o--o sequence
+    sequence o--o prove
+    P --> Chunk
+    P --> Batch
+    P --> Bundle
+```
+
+For more information about the Sindri platform, please check out [sindri.app](https://sindri.app/).
+The next section ([Getting Started](#getting-started)) shows how to launch a local Scroll SDK devote with provers running on Sindri.
+For production deployments, please consult the official [Scroll SDK documentation](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/production-deployment/).
+
+# Getting Started
+
+While this section primarily reiterates [Scroll's Devnet Guide](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/guides/devnet-deployment/), it includes adjustments specifically for Ubuntu environments.
+This guide also makes revisions to launch the coordinator service which acts as a gateway between the sequencing layer and Sindri provers.
+Since the coordinator requires at least 20 GB of RAM, we recommend you use a machine with at least 32 GB available.
+
+### Prerequisites
+
+You will need to obtain an API key to use Sindri.
+If you have not already created an account, you can do so by visiting the [Sindri sign up page](https://sindri.app/signup).
+After logging into the [Sindri front-end](https://sindri.app/login), you can create and manage your API Keys within the [API Keys Settings page](https://sindri.app/z/me/page/settings/api-keys).
```

Comment:
```suggestion
After logging into the [Sindri front-end](https://sindri.app/login), you can create and manage your API Keys within the [API Keys Settings page](https://sindri.app/login?next=/z/me/page/settings/api-keys).
```
By using the "next url" feature, this link will take you to your personal-team’s api key page upon login or it’ll take you to your currently selected team’s api-keys page if you are already logged in

Reply:
@KPreisner This link is labeled "API Keys Settings page". Do you want someone saving this link as the canonical api key page? Does our code not detect the "not logged in" event and redirect to login with "?next=..." automatically?

Reply:
[Our docs](https://sindri.app/docs/topic-guides/access-management/#api-key-creation-and-management) use a direct link to the api key settings page: `https://sindri.app/z/me/page/settings/api-keys`.

Our code does not detect the "not logged in" event and redirect to login with "?next=..." automatically. Maybe there's a good reason for that? In either case I'd go with our docs for consistency.